deploy/roles/scala/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+scala_version:              2.11.8
+scala_url:                  "http://downloads.lightbend.com/scala/{{ scala_version }}/scala-{{ scala_version }}.tgz"
+
+
```

Comment:
@olivermel Last thing - Can you clean up the newlines throughout this PR?  Use only 1 newline in between various logical sections of code, here you have 3, other places 2.  It makes everything easier to read.  Thanks!